debian/copyright

```diff
@@ -3,11 +3,12 @@ Upstream-Name: unvanquished
 Source: http://github.com/Unvanquished/Unvanquished
 
 Files: *
-Copyright: 2011-2013 Unvanquished Development (and others)
+Copyright: 2011-2021 Unvanquished Development (and others)
```

Comment:
2021 or 2022?

Reply:
Welp, I have set 2021 because the last release in changelog is from 2021, but on the other hand it also builds commits from 2022 so, maybe 2022.